service/controller/v24/resource/cloudformation/create.go

```diff
@@ -106,10 +105,6 @@ func (r *Resource) newCreateChange(ctx context.Context, obj, currentState, desir
 	if currentStackState.Name == "" || desiredStackState.Name != currentStackState.Name {
 		r.logger.LogCtx(ctx, "level", "debug", "message", "the guest cluster main stack has to be created")
 
-		if err := r.validateCluster(customObject); err != nil {
-			return cloudformation.CreateStackInput{}, microerror.Mask(err)
-		}
```

Comment:
I feel like this was completely useless.

Reply:
Maybe. It was added because of IPAM problems. The idea was if the VPC IP range is already in use to fail early rather than launch all the cloud formation stacks.

Not sure on the state of IPAM and whether this is something we still care about.

Reply:
IPAM should handle this.

Reply:
OK cool. I don't think we're having a lot of IPAM problems so I'm fine with this.